juniper/src/schema/model.rs

```diff
@@ -186,6 +219,11 @@ impl<'a, S> SchemaType<'a, S> {
             } else {
                 None
             },
+            subscription_type_name: if &subscription_type_name != "_EmptyMutation" {
```

Comment:
This should be `_EmptySubscription`

Reply:
I think we need one.
There are both [`EmptyMutation`](https://github.com/instrumentisto/juniper/blob/async-await-subscriptions/juniper/src/types/scalars.rs#L298) and `EmptySubscription` (can't see it on GitHub for some reason, will re-push later) in `src/types/scalars.rs`  

As I understood, these are used for schemas only with queries